### PR TITLE
Package renaming

### PR DESCRIPTION
**Pull request by jacobweinstock**
We moved a little too fast apparently and didn't name things as we should have. i believe a renaming like this will get much more difficult the more code we have. So here it is.

refs:
https://blog.golang.org/package-names
https://rakyll.org/style-packages/

### CODE REVIEW

File: pkg/config/stream/event.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package events
+package stream
```
Comment (bgentry-ntap) [inline pkg/config/stream/event.go:1]: This does not do what I would expect a stream package would do. How do you feel about 'progress' or 'status'? I was thinking we could also put the progress interfaces for the bootstrapper to monitor here.

Comment (j-griffith) [reply pkg/config/stream/event.go:1]: seems like it's streaming Event objects no?

Comment (bgentry-ntap) [reply pkg/config/stream/event.go:1]: There is no streaming happening in this package. This is just a DTO which is probably a bad pattern to begin with. All the clients happen to use this to report progress over the events chan but that is all outside this package. Maybe the best solution is to create a separate package for events/progress that also contains the logic for that.

**Pull request merged**